script/cms/compare-json.js

```diff
@@ -182,24 +225,48 @@ const compareJson = (baseGraphQlObject, baseCmsExportObject) => {
     }
   });
 
+  const arraysToCompare = [];
+
   // Get array of differences. Exclude new properties because transformed
   // CMS objects may have additional properties
-  return deepDiff(graphQlObject, cmsExportObject, (diffPath, key) => {
+  const diffs = deepDiff(graphQlObject, cmsExportObject, (diffPath, key) => {
+    // Store arrays for comparison by alternate method.
+    const cmsObject = Object.byArray(cmsExportObject, diffPath);
+    const graphQLObject = Object.byArray(cmsExportObject, diffPath);
```

Comment:
Shouldn't this be:
```suggestion
    const graphQLObject = Object.byArray(graphQlObject, diffPath);
```
Otherwise `cmsObject` and `graphQLObject` will always be the same value.

Reply:
Arrrrgh! Good catch, thanks!